Screenbox/Pages/SettingsPage.xaml

```diff
@@ -315,6 +315,13 @@
                     <ToggleSwitch AutomationProperties.HelpText="{x:Bind SettingsEnqueueAllFilesInFolderCard.Description}" IsOn="{x:Bind ViewModel.EnqueueAllFilesInFolder, Mode=TwoWay}" />
                 </ctc:SettingsCard>
 
+                <ctc:SettingsCard
+                    Margin="{StaticResource SettingsCardMargin}"
+                    Description="{strings:Resources Key=SettingsRestorePlaybackPositionDescription}"
+                    Header="{strings:Resources Key=SettingsRestorePlaybackPositionHeader}">
```

Comment:
What do you think about using the replay icon?

```suggestion
                    Header="{strings:Resources Key=SettingsRestorePlaybackPositionHeader}"
                    HeaderIcon="{ui:FontIcon Glyph=&#xEF3B;}">
```

Reply:
That's not an ideal icon for this option, but it should be good enough for now. We should consider what to do when we have more specific options that we can't find an icon for. Currently, when options get too specific, we group them into a SettingsExpander and give the expander an icon, like the Gestures options. Is this a good enough solution or do we need something else?

Reply:
I think it effectively conveys the feature. The anti-clockwise arrow is a good starting point and base, what else should we add to improve it?

For the rest, we should first explore the fluent icons document for inspiration. If we can't find a suitable icon, we should look for general ideas and adapt them to "our" style

Reply:
I think the idea of "playback position" is lost in this icon specifically. However, it's not a blocker.

For the rest, I understand that we can custom-craft our own icons for each option. The question is: Should we? Do we really need each option to have its own icon? If you dig deep enough in the Windows Settings, you will see pages with a mix of icon and no-icon options. I think we should embrace no-icon options or hide them somehow.

> Currently, when options get too specific, we group them into a SettingsExpander and give the expander an icon, like the Gestures options.

This is a way to hide no-icon options.

Reply:
> I think the idea of "playback position" is lost in this icon specifically. However, it's not a blocker.

I can see that, resume vs. replay.

> For the rest, I understand that we can custom-craft our own icons for each option. The question is: Should we? Do we really need each option to have its own icon?

For top-level options an icon should be required.

> If you dig deep enough in the Windows Settings, you will see pages with a mix of icon and no-icon options. I think we should embrace no-icon options or hide them somehow.

I agree, certain options within a subpage (maybe even a group) may not require an icon.